### ISSUE TITLE
  - Type 'CloseAndReleaseRepositoryTask' property 'nexusOptions.stagingProfile' doesn't have a configured value.

### ISSUE THREAD

**Issue by marandaneto** [vanniktech/gradle-maven-publish-plugin#234]
Plugin: 0.14.2 (tried with 0.15.0 too)
Gradle: 6.8.3
OS: macOS 11.2.3

when executing: `:closeAndReleaseRepository`

> A problem was found with the configuration of task ':closeAndReleaseRepository' (type 'CloseAndReleaseRepositoryTask').
>   - Type 'CloseAndReleaseRepositoryTask' property 'nexusOptions.stagingProfile' doesn't have a configured value.
>     
>     Reason: This property isn't marked as optional and no value has been configured.
>     
>     Possible solutions:
>       1. Assign a value to 'nexusOptions.stagingProfile'.
>       2. Mark property 'nexusOptions.stagingProfile' as optional.

to fix, you've to set `stagingProfile` eg

```
val publish = extensions.getByType(MavenPublishPluginExtension::class.java)
publish.releaseSigningEnabled = BuildUtils.shouldSignArtifacts()
publish.nexus {
    stagingProfile = group.toString()
}
```

`group` is set in the gradle.properties though.

Might be related to https://github.com/vanniktech/gradle-maven-publish-plugin/issues/202

**Comment by gabrielittner** [comment]
Which version of the plugin are you using?

**Comment by marandaneto** [comment]
@gabrielittner  updated the description

**Comment by gabrielittner** [comment]
This should be fixed in 0.15.1 which was just released

**Comment by marandaneto** [comment]
its fixed, thanks
